faq.md

```diff
@@ -12,8 +12,7 @@ The final text of the CRA can be found on [EUR-Lex][CRA] ([English HTML version]
 
 #### Q: When does the CRA enter into force and when does the regulation start to apply?
 
-The CRA enters into force on December 11, 2024. Reporting obligations of manufacturers ([Article 14][]) start to apply on September 11, 2026.
-The notification of conformity of assement bodies ([Chapter IV][]) start to apply on June 11, 2026. Everything else starts to apply on December 11, 2027.
+The CRA enters into force on December 11, 2024. Reporting obligations of actively exploited vulnerabilities and severe incidents ([Article 14][]) start to apply on September 11, 2026. The notification of conformity of assement bodies ([Chapter IV][]) start to apply on June 11, 2026. Everything else starts to apply on December 11, 2027. _(Source: [Article 71][])_
```

Comment:
This opens for some more questions to the FAQ:

*who* is obliged to report about "vulnerabilities and severe incidents", to whom and how is it supposed to be done?

Reply:
"exploited vulnerabilities", that changes quite much

Reply:
Transferred @bagder 's question to another ticket.